app/src/api/campaigns.ts

```diff
@@ -0,0 +1,46 @@
+import request from '@/utils/request';
+
+import { AxiosPromise } from 'axios';
+import { ICampaigns } from './types';
+
+export const defaultCampaignData: ICampaigns = {
+  name: '',
+  startAt: 1,
+  endAt: 1,
```

Comment:
StartAt and EndAt should be at 0 if the haven't been defined (0 => null timestamp), 1 in timestamp correspond to '1/1/1970 à 0:00:01'